### PR TITLE
Ar feature additional packages

### PR DESCRIPTION
**Pull request by aroerick**
Add vector icons package, previous onboarding package was no longer maintained and gave proptype issues.

### CODE REVIEW

File: js/App.js [hunk 1]
```diff
@@ -12,7 +13,8 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text>Welcome to AssistList</Text>
+        <LoginContainer />
+        {/* <Text>Welcome to AssistList</Text> */}
```
Comment (10thfloor) [inline js/App.js:17]: Be careful. If you're not planning on using this commented out code, please remove it from your PR.

File: js/components/AccountInput/AccountInput.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { TextInput } from 'react-native';
+import styles from './styles';
+
+export const AccountInput = ({ input, name, type }) => {
+  return (
+    <TextInput
+      style={styles.input}
+      placeholder={name}
+      placeholderTextColor="#00ADCB80"
```
Comment (10thfloor) [inline js/components/AccountInput/AccountInput.js:10]: This is not a valid Hex code.

File: js/screens/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { Fragment } from 'react';
+import { Text, View, TextInput, TouchableOpacity } from 'react-native';
+import { Form, Field } from 'react-final-form';
+import { AccountInput } from '../../components/AccountInput/AccountInput';
+
+const Login = () => {
+  return (
+    <Form
+      onSubmit={values => console.log(values)}
```
Comment (10thfloor) [inline js/screens/Login/Login.js:9]: Please remove console.log calls from your PRs.

File: js/screens/Login/Login.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      render={({ handleSubmit, pristine, values }) => (
+        <Fragment>
+          <Field name="email">
+            {({ input, meta }) => <AccountInput input={input} name="Email" />}
```
Comment (10thfloor) [inline js/screens/Login/Login.js:13]: What is the purpose of the name attribute here?

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: js/App.js [hunk 5]
```diff
@@ -12,7 +13,8 @@ export default class App extends Component {
...
       </View>
     );
   }
```
Comment (siddharthparmar7) [inline js/App.js:20]: Please remove the styles from here.

File: js/App.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import { Platform, StyleSheet, Text, View } from 'react-native';
```
Comment (siddharthparmar7) [inline js/App.js:2]: Please remove unnecessary imports.

File: js/App.js [hunk 7]
```diff
@@ -1,5 +1,6 @@
...
+import LoginContainer from './screens/Login/';
 
 const instructions = Platform.select({
```
Comment (siddharthparmar7) [inline js/App.js:5]: please remove instructions.

File: js/App.js [hunk 8]
```diff
@@ -12,7 +13,8 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
```
Comment (siddharthparmar7) [inline js/App.js:15]: Remove this View component if not needed.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Pull request closed without merging**